<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QMainWindow" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditorWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Monospace&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string/>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionReplace"/>
    <addaction name="actionGo_To"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionTime_Date"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string/>
    </property>
    <addaction name="actionWord_Wrap"/>
    <addaction name="separator"/>
    <addaction name="actionFont_Style"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string/>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPage_Setup"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As"/>
  <action name="actionUndo">
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionWord_Wrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFont_Style"/>
  <action name="actionPage_Setup"/>
  <action name="actionPrint">
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit"/>
  <action name="actionDelete">
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionGo_To">
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionTime_Date">
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>actionCopy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>actionCut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>redoAvailable(bool)</signal>
   <receiver>actionRedo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>actionUndo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>EditorWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_All</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
